uiTests/src/test/java/com/virtuslab/gitmachete/uitest/UITestSuite.java

```diff
@@ -79,22 +79,27 @@ public void skipNonExistentBranchesAndToggleListingCommits() {
     int branchAndCommitRowsCount = callJs("project.refreshGraphTableModel().getRowCount()");
     // 6 branch rows + 7 commit rows
     Assert.assertEquals(13, branchAndCommitRowsCount);
+
+    // Let's slide out a root branch now
+    runJs("project.slideOutBranch('develop')");
+    awaitIdle();
+    branchAndCommitRowsCount = callJs("project.refreshGraphTableModel().getRowCount()");
+    // 5 branch rows (`develop` is no longer there) + 3 commit rows
+    // (1 commit of `allow-ownership-link` and 3 commits of `call-ws` are all gone)
+    Assert.assertEquals(8, branchAndCommitRowsCount);
```

Comment:
*TODO:* this will now fail (8 != 13) since sliding out roots isn't allowed (which proved to be rather random&unjustified to our users)